andela/ah-scorpion-frontend
# #159993327 Set up React using create-react-app

**What does this PR do?**

- Set up React using create-react-app.
- Set up Airbnb linting.
- Sets up Bootstrap v4 for the project.

**Description of tasks completed**

- Sets up React using create-react-app.
- Sets up Airbnb linting.
- Sets up Bootstrap v4 for the project.

**How can this PR be manually tested?**

- Clone this branch.
- Run `npm install` to install dependencies.
- Run `npm run test` to run the existing test. The test should pass.
- Run `npm start` and you should see a React welcome page on your favourite browser.

**What are the relevant pivotal tracker stories?**

[#159993327](https://www.pivotaltracker.com/story/show/159993327)
